src/test/actions/authActions/resetPassword.spec.js

```diff
@@ -0,0 +1,63 @@
+import { shallow } from 'enzyme';
+import sinon from 'sinon';
+import React from 'react';
+import { ResetPasswordCard } from '../../../components/ResetPassword/ResetPassword';
+
+
+describe('Test methods', () => {
+  beforeEach(() => {
+    sinon.spy(ResetPasswordCard.prototype, 'handleSubmit');
+    sinon.spy(ResetPasswordCard.prototype, 'handleEmail');
+  });
+  it('should tes', () => {
+    expect(2).toBe(2);
```

Comment:
@beejay1293 Please is there a reason why this assertion is necessary in your test file? As shown here, It is asserting for `2 to be 2`